src/core/server/security/types.ts

```diff
@@ -3,11 +3,12 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import { Capabilities, OpenSearchDashboardsRequest } from '../index';
+import { Capabilities, OpenSearchDashboardsRequest, RequestHandlerContext } from '../index';
 
 export interface SecurityServiceSetup {
   registerReadonlyService(service: IReadOnlyService): void;
   readonlyService(): IReadOnlyService;
+  registerSourceHandler(source: string, handler: IdentitySourceHandler): void;
```

Comment:
```suggestion
  registerIdentitySourceHandler(source: string, handler: IdentitySourceHandler): void;
```
`registerIdentitySourceHandler` seems more clear to me.

Reply:
updated